recipes/ratingcurve/LICENSE.md

```diff
@@ -0,0 +1,46 @@
+License
```

Comment:
You should remove this license file since it is already included in the tarball on pypi.